cmd/suseconnect/suseconnect.go

```diff
@@ -43,6 +44,10 @@ func (p *singleStringFlag) Set(value string) error {
 }
 
 func main() {
+	if runtime.GOOS != "linux" {
```

Comment:
can you add an comment linking https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63 so next person looking at this has the information at hand?

Reply:
Done. I've linked the Go's source code directly.